modules/deployment/main.tf

```diff
@@ -120,6 +120,30 @@ resource "aws_codepipeline" "this" {
       }
     }
   }
+
+  # add abritary post deployment steps including approval stages
+  dynamic "stage" {
```

Comment:
Could you enhance the [complete example](https://github.com/moritzzimmer/terraform-aws-lambda/tree/main/examples/deployment/complete) with a simple additional stage to showcase/test this feature?